### ISSUE TITLE
Issue in return value of textRequestHandler

### ISSUE THREAD

**Issue by pgibler** [typpo/textbelt#99]
https://github.com/typpo/textbelt/blob/master/server/app.js#L128

"succes" needs to be replaced with with "success" so clients don't barf on this condition.

**Comment by pgibler** [comment]
I've spent part of today looking into why this is happening and haven't gotten any further.

When I try to directly send an email from mutt, I also don't have receive the text message. The command I used is as follows:

`mutt -s txt -e set from=test_work@gmail.com -e set user_from=yes -e set envelope_from=yes -b 5082591617@vtext.com`

I don't know if it's a carrier issue, as it doesn't work when using the standard mode of operation (not specifying carrier) or if I specify carrier (ATT & Verizon were both tested).

Is there some networking issue that could be happening here? I'm running textbelt on an Ubuntu 16.04 machine in a VirtualBox VM. Given that it has full internet access and port 9090 is forwarded, I'm not sure what else would be required from a network point of view to get this working.

If anyone has any insights, I'd be very appreciative. I'll keep looking into this.
